train_dolly.py

```diff
@@ -179,10 +192,12 @@
 
 from training.generate import generate_response, load_model_tokenizer_for_generate
 
-model, tokenizer = load_model_tokenizer_for_generate(local_output_dir)
+model, tokenizer = load_model_tokenizer_for_generate(dbfs_output_dir)
```

Comment:
I made this change because during testing my cluster would auto-terminate (training takes about 6 hours with 4 V100 GPUs). The local output dir disappears when the cluster shuts down, but the DBFS location remains. Since we're copying the data to DBFS, I figured why not leave it this way.

Reply:
Makes sense